text/0006-monolothic-packaging.md

```diff
@@ -279,6 +279,35 @@ Open issues:
 - Should we use `aws_cdk` as the module name to preserve compatibility or rename
   to `aws_cdk_lib`?
 
+### Go
+
+Usage:
+
+```go
+import (
+    "github.com/aws/aws-cdk-go/awscdk"
+)
+
+app := awscdk.NewApp()
+stack := awscdk.NewStack(app, "MyStack");
+bucket := awscdk.aws_s3.NewBucket(stack, "MyBucket")
```

Comment:
Right now I believe this will be `awscdk.awss3.NewBucket`. Package names are just all lowered.

Reply:
Yes, you are correct. are we okay with that? Looks okay to me...

![Screen Shot 2021-03-04 at 20 10 21](https://user-images.githubusercontent.com/598796/110009410-a75bb800-7d25-11eb-956d-22b7fdaf1226.png)

Reply:
yeah I'm good with it, I think for package names this is more Go-ey.

Reply:
Fixed